src/schemas/components/SchemaFormPropertyString.vue

```diff
@@ -70,9 +61,10 @@
       })
     }
 
-    return withProps(PTextInput, {
+    return withProps(PTextarea, {
```

Comment:
I forget how we treat this - do we need to restrict resizing at all?

Reply:
Its 1 row by default but allowed to resize. And PTextarea only allows vertical resizing so it doesn't overflow the layout.

Reply:
ah nice ok - thanks for confirming, that's exactly what i was hoping for.